src/dialects/db2/index.ts

```diff
@@ -0,0 +1,78 @@
+import type { Sequelize } from '../../sequelize.js';
+import { createUnspecifiedOrderedBindCollector } from '../../utils/sql';
+import { AbstractDialect } from '../abstract';
+import { Db2ConnectionManager } from './connection-manager';
+import * as DataTypes from './data-types.js';
+import { Db2Query } from './query';
+import { Db2QueryGenerator } from './query-generator';
+import { Db2QueryInterface } from './query-interface';
+
+export class Db2Dialect extends AbstractDialect {
+  static supports = AbstractDialect.extendSupport({
+    migrations: false,
+    schemas: true,
+    finalTable: true,
+    autoIncrement: {
+      defaultValue: false,
+    },
+    alterColumn: {
+      unique: false,
+    },
+    index: {
+      collate: false,
+      using: false,
+      where: true,
+    },
+    tmpTableTrigger: true,
+    dataTypes: {
+      COLLATE_BINARY: true,
+      TIME: {
+        precision: false,
+      },
+    },
+    milliseconds: true,
+  });
+
+  readonly defaultVersion = '1.0.0';
+  readonly dataTypesDocumentationUrl = 'https://www.ibm.com/support/knowledgecenter/SSEPGG_11.1.0/com.ibm.db2.luw.sql.ref.doc/doc/r0008478.html';
+  readonly connectionManager: Db2ConnectionManager;
+  readonly queryGenerator: Db2QueryGenerator;
+  readonly queryInterface: Db2QueryInterface;
+  readonly Query = Db2Query;
+
+  /** @deprecated */
```

Comment:
```
  /** @deprecated */
  readonly TICK_CHAR = '"';
  readonly TICK_CHAR_LEFT = '"';
  readonly TICK_CHAR_RIGHT = '"';
 ```

Is there any link about the deprecation?

Reply:
A link to what? :o

Reply:
> A link to what? :o

The information why that is deprecated but nevermind I can check it later